### DISCUSSION TITLE
Very strange

### DISCUSSION THREAD

**Discussion by Adedoyin-Emmanuel** [fasakinhenry/fasakinhenry#5] [Q&A] [answered]
Man, why on earth will you add Github actions on your Github profile README?

**Comment by fasakinhenry** [comment, marked as answer]
I used the github actions for my gitartwork and youtube cards on my profile readme

**Reply by Adedoyin-Emmanuel** [reply]
Cool stuff

**Reply by Adedoyin-Emmanuel** [reply]
Cool stuff
